__tests__/users.tests.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  connection.end();
+});
+
+describe("GET api/users", () => {
+  test("GET - status: 200 - responds with an array of user objects ", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { users } = response.body;
+
+        expect(users.length).toBe(4);
+        users.forEach((user) => {
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
+      });
+  });
+});
+
+describe("GET api/users", () => {
+  test("GET - status: 404 - when provided non existent endpoint", () => {
```

Comment:
This 404 is probably covered in your earlier tests for invalid end point, but you can leave this in if you want :)